JavaScript/JavaScriptSDK/Telemetry/PageView.ts

```diff
@@ -21,9 +22,10 @@ module Microsoft.ApplicationInsights.Telemetry {
         /**
          * Constructs a new instance of the PageEventTelemetry object
          */
-        constructor(name?: string, url?: string, durationMs?: number, properties?: any, measurements?: any) {
+        constructor(id?: string, name?: string, url?: string, durationMs?: number, properties?: any, measurements?: any) {
```

Comment:
Update API reference doc in this repository as well.

Reply:
id was already mentioned as a field for telemetry item... no need to update.

Reply:
optional parameters should be added at the end?

Reply:
yes, i already updated it - you're looking at the outdated version )

Reply:
ah, thanks!